src/main/java/org/truffleruby/interop/InteropNodes.java

```diff
@@ -75,22 +78,20 @@ public Object importFile(DynamicObject fileName) {
 
     }
 
+    @ImportStatic(Message.class)
     @CoreMethod(names = "executable?", isModuleFunction = true, required = 1)
     public abstract static class IsExecutableNode extends CoreMethodArrayArgumentsNode {
 
         @Specialization
         public boolean isExecutable(
                 TruffleObject receiver,
-                @Cached("createIsExecutableNode()") Node isExecutableNode) {
+                @Cached("IS_EXECUTABLE.createNode()") Node isExecutableNode) {
             return ForeignAccess.sendIsExecutable(isExecutableNode, receiver);
         }
 
-        protected Node createIsExecutableNode() {
-            return Message.IS_EXECUTABLE.createNode();
-        }
-
     }
 
+    @ImportStatic(Message.class)
```

Comment:
This ImportStatic is not used, bu it doesn't hurt either.

Reply:
Fixed.